src/Views/AppListUpdateView.vala

```diff
@@ -122,7 +122,7 @@ namespace AppCenter.Views {
             var needs_update = package.state == AppCenterCore.Package.State.UPDATE_AVAILABLE;
 
             // Only add row if this package needs an update or it's not a font or plugin
-            if (needs_update || (!package.is_plugin && !package.is_font)) {
+            if (needs_update || (!(package.kind == AppStream.ComponentKind.ADDON) && !(package.kind == AppStream.ComponentKind.FONT))) {
```

Comment:
Isn't `package.kind != AppStream.ComponentKind.ADDON`  etc a bit clearer than `!(package.kind == AppStream.ComponentKind.ADDON)`?

Reply:
omg yeah, why did I write it like that lmao